lib/context-menu-handlers.js

```diff
@@ -105,15 +105,32 @@ function init(panel) {
     }
   });
 
+  const contentScriptVimeoCase = `self.on('click', function (node, data) {
+    self.postMessage('https://vimeo.com/' + node.href);
+  });`
+
   cm.Item({
     label: contextMenuLabel,
     context: [
       cm.URLContext(['*.vimeo.com']),
       cm.SelectorContext('.contextclip-header a')
     ],
-    contentScript: `self.on('click', function (node, data) {
-      self.postMessage('https://vimeo.com/' + node.href);
-    });`,
+    contentScript: contentScriptVimeoCase,
+    onMessage: (url)=> {
```

Comment:
This function is a dupe of the `onMessage` function that starts at line 134, would be good to extract it too